src/server/utils/bucket_diff.js

```diff
@@ -209,8 +222,10 @@ class BucketDiff {
         dbg.log2('BucketDiff get_objects:: bucket_response', list_objects_response);
         const bucket_contents_left = this._object_grouped_by_key_and_omitted(list_objects_response);
         const bucket_cont_token = this._get_next_key_marker(list_objects_response, bucket_contents_left);
-        dbg.log2('BucketDiff get_objects:: bucket', bucket, 'bucket_contents_left', bucket_contents_left);
-        return { bucket_contents_left, bucket_cont_token };
+        const field = 'DeleteMarker';
```

Comment:
This will fail on a bucket that have no version enabled.

Reply:
Thanks, I will look into it.